superset/connectors/base/models.py

```diff
@@ -185,6 +191,8 @@ def data(self):
             'metrics': [o.data for o in self.metrics],
             'columns': [o.data for o in self.columns],
             'verbose_map': verbose_map,
+            'schema': self.schema,
+            'select_star': self.select_star,
```

Comment:
Given this here in the base class I'd go for a `pass`  or `return None` in `BaseDatasource.select_star`